generate-docs/tools/package-lock.json

```diff
@@ -1,1809 +1,8 @@
 {
   "name": "tools",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Not a big deal, but double check that you have the latest released version of Node.